docs/reference/faq.md

```diff
@@ -45,45 +45,153 @@ Our goal with Gutenberg is not just to create a seamless post- and page-building
 
 Yes. There are a lot! There is a help modal showing all available keyboard shortcuts.
 
-You can see the whole list going to the top right corner menu of the new editor and clicking on "Keyboard Shortcuts" (or by using the keyboard shortcut Ctrl + Option + H).
+You can see the whole list going to the top right corner menu of the new editor and clicking on “Keyboard Shortcuts” (or by using the keyboard shortcut <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>H</kbd> on Windows/Linux and <kbd>⌃</kbd><kbd>⌥</kbd><kbd>H</kbd> on macOS).
 
 This is the canonical list of keyboard shortcuts:
 
-### Editor shortcuts
-
-* <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>H</kbd> Display keyboard shortcuts.
-* <kbd>⌘S</kbd> Save your changes.
-* <kbd>⌘Z</kbd> Undo your last changes.
-* <kbd>Shift</kbd>+<kbd>⌘Z</kbd> Redo your last undo.
-* <kbd>Shift</kbd>+<kbd>⌘,</kbd> Show or hide the settings sidebar.
-* <kbd>Ctrl</kbd>+<kbd>\`</kbd> Navigate to a the next part of the editor.
-* <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>\`</kbd> Navigate to the previous part of the editor.
-* <kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>N</kbd> Navigate to a the next part of the editor (alternative).
-* <kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>P</kbd> Navigate to the previous part of the editor (alternative).
-* <kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>⌘M</kbd> Switch between Visual Editor and Code Editor.
-
-### Selection shortcuts
-
-* <kbd>⌘A</kbd> Select all text when typing. Press again to select all blocks.
-* <kbd>Esc</kbd> Clear selection.
-
-### Block shortcuts
-
-* <kbd>Shift</kbd>+<kbd>⌘D</kbd> Duplicate the selected block(s).
-* <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>z</kbd> Remove the selected block(s).
-* <kbd>Option</kbd>+<kbd>⌘T</kbd> Insert a new block before the selected block(s).
-* <kbd>Option</kbd>+<kbd>⌘Y</kbd> Insert a new block after the selected block(s).
-* <kbd>/</kbd> Change the block type after adding a new paragraph.
-
-### Text formatting
-
-* <kbd>⌘B</kbd> Make the selected text bold.
-* <kbd>⌘I</kbd> Make the selected text italic.
-* <kbd>⌘U</kbd> Underline the selected text.
-* <kbd>⌘K</kbd> Convert the selected text into a link.
-* <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>S</kbd> Remove a link.
-* <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>D</kbd> Add a strikethrough to the selected text.
-* <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>X</kbd> Display the selected text in a monospaced font.
+<table>
+	<thead>
+		<tr>
+			<th>Shortcut description</th>
+			<th>Windows/Linux shortcut</th>
+			<th>macOS shortcut</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<th colspan="3">Editor shortcuts</th>
```

Comment:
Not sure this is the best way to use a table header. Screen readers do announce the table headers (th) when navigating through the table cells. While the first 3 headers in the `<thead>` section give a proper information, I'm not sure what happens when then there's another header with `colspan="3"`.

I'd suggest to separate sections with headings and use separate tables.

Testing with just Safari and VoiceOver isn't enough, this should be tested also on Windows (at least) with JAWS and NVDA.

Rather than abstract semantics, I'd suggest to keep things as simple as possible, ensuring information is announced correctly an navigable with ease. Also, redundant information and "noise" for screen readers should be avoided. For example. all the "plus" get announced and they're a bit annoying, not strictly relevant.

I'd also suggest to have a look at how webaim.org made similar tables, e.g.:
https://webaim.org/resources/shortcuts/jaws
https://webaim.org/resources/shortcuts/nvda

Reply:
> I'd also suggest to have a look at how webaim.org made similar tables, e.g.:
https://webaim.org/resources/shortcuts/jaws
https://webaim.org/resources/shortcuts/nvda

I find the tables in those links really hard to parse with the lack of consistent indentation, line breaks mid-way through shortcuts and highly variable column widths. I'm very surprised if that's the gold standard for accessibility.

Reply:
![screen shot 2018-10-12 at 09 44 17](https://user-images.githubusercontent.com/699132/46855291-849e7100-ce03-11e8-8213-17da7d113d0f.png)

I observed that `keycap` class shares styles with `kbd`. I bet you can simplify those tables and move on.

Reply:
Yeah, I'd been keen to just get this merged now, separate tables or not. Will probably need iteration in a second PR or trac ticket once we know how it looks in the handbook.

Reply:
> For example. all the "plus" get announced and they're a bit annoying, not strictly relevant.

Let's remove it as suggested.

Reply:
> Let's remove it as suggested.

They're that way in the Gutenberg front-end, though. If we remove them here, lets remove them in Gutenberg as well. I'd prefer it if we kept things consistent so that if we apply an improvement in once place we apply it in others as well. Maybe we can get this merged with the 'plus' symbols in place and then create an issue where we can discuss removing them?

Reply:
Good call @talldan, ideally we could copy and paste HTML from the modal dialog to the handbook to make the process easier.